test/fail_compilation/staticarrayoverflow2.d

```diff
@@ -0,0 +1,10 @@
+/*
+REQUIRED_ARGS: -m64

```

Comment:
@NightmareX1337 if I understand you correctly, this PR changes things from issuing errors in the front end to crashing in the back end? Also, I don't see where things like `byte[size_t.max]` would ever be accepted by the front end.